main/mariadb/APKBUILD

```diff
@@ -40,16 +40,17 @@ fi
 source="https://downloads.mariadb.org/interstitial/mariadb-$pkgver/source/mariadb-$pkgver.tar.gz
 	$pkgname.initd
 	fix-c11-atomics-check.patch
-	fix-mysql-install-db-path.patch
```

Comment:
I don't understand why this patch was removed. Can you help me?

Reply:
This has been fixed upstream: https://jira.mariadb.org/browse/MDEV-14580